HW 1/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Homework</title>
+</head>
+<body>
+
+    <button id="myBtn"> Try it </button>
+    
+    <button class="myTogBtn"> Toggle Button </button>

```

Comment:
myTogBtn is not a really proper name for class or something else. Try to not use "my" in any namings (except name for your website or page). Better could be: toggleBtn, toggleButton

Reply:
Yes teacher, I remember that.